.github/workflows/update-dotnet-install-script.yml

```diff
@@ -5,7 +5,8 @@ on:
     - cron:  '0 4 1 * *'
   workflow_dispatch:
 
-permissions: {}
+permissions:
+  contents: read
```

Comment:
This is fine, the script just didn't understand it, I'll close this PR and make the script smarter next time